core/controllers/topic_editor.py

```diff
@@ -77,7 +77,7 @@ def get(self, topic_id):
                 Exception('The topic with the given id doesn\'t exist.'))
 
         self.values.update({
-            'topic_id': topic.id
+            'topicId': topic.id
```

Comment:
Follow existing convention of using snake_case for dicts coming from the backend. (When constructing the domain object in the frontend, convert these snake_case names to camelCase.)

Reply:
Done.